/admin is broken in develop

Caused by 5c9d081391183deee1eccccea88d194ae5952341

```
[2013-02-28 01:36:21.243] [ERROR] console - Error: Unable to copy obj! Its type isn't supported.
    at clone (/home/jose/etherpad-lite/src/node_modules/ueberDB/CloneAndAtomicLayer.js:209:9)
    at clone (/home/jose/etherpad-lite/src/node_modules/ueberDB/CloneAndAtomicLayer.js:204:50)
    at exports.database.set (/home/jose/etherpad-lite/src/node_modules/ueberDB/CloneAndAtomicLayer.js:72:79)
    at SessionStore.set (/home/jose/etherpad-lite/src/node/db/SessionStore.js:38:6)
    at Session.save (/home/jose/etherpad-lite/src/node_modules/express/node_modules/connect/lib/middleware/session/session.js:63:25)
    at ServerResponse.res.end (/home/jose/etherpad-lite/src/node_modules/express/node_modules/connect/lib/middleware/session.js:280:19)
    at ServerResponse.res.send (/home/jose/etherpad-lite/src/node_modules/express/lib/response.js:149:8)
    at exports.expressCreateServer (/home/jose/etherpad-lite/src/node/hooks/express/adminplugins.js:18:9)
    at callbacks (/home/jose/etherpad-lite/src/node_modules/express/lib/router/index.js:161:37)
    at param (/home/jose/etherpad-lite/src/node_modules/express/lib/router/index.js:135:11)
```

Caused by https://github.com/ether/etherpad-lite/commit/5c9d081391183deee1eccccea88d194ae5952341#L2R105 but changing back breaks reconnect functionality :|

--- a/src/node/utils/Settings.js
+++ b/src/node/utils/Settings.js
@@ -157,6 +157,7 @@ exports.reloadSettings = function reloadSettings() {
   try {
     if(settingsStr) {
       settings = vm.runInContext('exports = '+settingsStr, vm.createContext(), "settings.json");
+      settings = JSON.parse(JSON.stringify(settings)) // fix objects having constructors of other vm.context
     }
   }catch(e){
     console.error('There was an error processing your settings.json file: '+e.message);

